src/dsp_tools/utils/connection_live.py

```diff
@@ -110,6 +179,7 @@ def _write_request_to_file(
         jsondata: Optional[str],
         params: Optional[dict[str, Any]],
         response: requests.Response,
+        filepath: Optional[str] = None,
```

Comment:
In a method named "_write_request_to_file", a parameter named "filepath" suggests that you write to that file. Better name suggestion: "uploaded_file"

Reply:
you're right, fixed